pages/index.js

```diff
@@ -243,35 +321,144 @@ export default function Home() {
             zIndex: '20'
           }} className={styles.innerContent}>
           <h2>What's Hack OC?</h2>
-          <p>Hack OC is the first in-person high school hackathon after the pandemic in Orange County, California. We're inviting you and all high schoolers to participate in <span>12 hours</span> of <span>coding</span>, <span>building</span>, <span>learning</span>, and <span>sharing</span>. Whether you're technical and experienced or haven't ever written a line of code, Hack OC will be a fun and welcoming event for everyone.</p>
+          <p>Hack OC is the first in-person high school hackathon after the pandemic in Orange County, California. We're inviting all high schoolers to participate in <span>12 hours</span> of <span>coding</span>, <span>building</span>, <span>learning</span>, and <span>fun</span>. Whether you're technical and experienced or haven't ever written a line of code, Hack OC will be a fun and welcoming event for everyone.</p>
+
+          </div>
+          </div>
+                  {/* 3 columns on large screens, 2 on medium, 1 on small */}
+
+        <div style={{
+          display: 'grid',
+          gridTemplateColumns: 'repeat(auto-fit, minmax(min(' + cardSize + ', 100%), 1fr))',
+          gridGap: cardGap,
+          padding: '30px',
+          margin: '0 auto',
+          maxWidth: `calc(calc(${cardSize} * 2) + ${cardGap} + 60px + 200px)`
+        }}>
+          
+            <Card title="Build something" icon="idea">
+              By the end, you’ll have built something new and unique with your team.
+            </Card>
+            <ImageCard src="/social.png" caption="Placeholder image" />
```

Comment:
Do we want to keep the placeholder image caption

Reply:
Just fixed!